shark/iree_eager_backend.py

```diff
@@ -0,0 +1,73 @@
+from typing import Dict, Any, TypeVar
+
+import iree
+import iree.runtime as ireert
+import numpy as np
+import torch
+from iree.runtime import DeviceArray
+from torch_mlir._mlir_libs._mlir.ir import Module
+from torch_mlir._mlir_libs._mlir.passmanager import PassManager
+from torch_mlir_e2e_test.eager_backends.refbackend import NUMPY_TO_TORCH_DTYPE_DICT
+from torch_mlir.eager_mode.torch_mlir_eager_backend import (
+    TensorMetaData,
+    TorchMLIREagerBackend,
+)
+
+from shark.iree_utils import get_iree_compiled_module, IREE_DEVICE_MAP
+
+
+def get_stride_from_shape(shape, column_major=True):
```

Comment:
I dont see this function used..

Reply:
good catch - yea that's from before my refactor of the `add_device` torch-mlir pr.